browser/css/toolbar.css

```diff
@@ -380,9 +380,9 @@ td[id^='tb_editbar_item_sidebar']{
 	margin: 0 !important;
 }
 
-.w2ui-tb-image {
-	width: 24px !important;
-	height: 24px !important;
+.w2ui-toolbar table.w2ui-button .w2ui-icon {
+	width: var(--icon-size);
+	height: var(--icon-size);
```

Comment:
Thanks, but this seems to no only change to css vars but also going from affecting only related children from w2ui-toolbar to every single w2ui-tb-image no matter the parent and even if that are not a w2ui-icon. Was this intended?